Sources/Controllers/MessagesViewController+Keyboard.swift

```diff
@@ -45,9 +45,15 @@ internal extension MessagesViewController {
 
     @objc
     private func handleTextViewDidBeginEditing(_ notification: Notification) {
-        if scrollsToBottomOnKeyboardBeginsEditing {
-            guard let inputTextView = notification.object as? InputTextView, inputTextView === messageInputBar.inputTextView else { return }
-            messagesCollectionView.scrollToBottom(animated: true)
+        if scrollsToLastItemOnKeyboardBeginsEditing || scrollsToBottomOnKeyboardBeginsEditing {
```

Comment:
hmm, should this be an enum with top/topItem as well (In case an end-user is doing chat bottom to top)?

Reply:
How about we decide on one of these 2 approaches first, then extending to top? Unless you need scroll to top right now.